grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -397,6 +397,18 @@ class ProductAvailabilityService {
         return quantityOnHand
     }
 
+    def getQuantityNotPickedInLocation(Product product, Location location) {
+        def quantityOnHand = ProductAvailability.createCriteria().get {
+            projections {
+                sum("quantityNotPicked")
+            }
+            eq("product", product)
+            eq("location", location)
+        }
+
+        return quantityOnHand
```

Comment:
this should be quantityNotPicked instead of quantityOnHand

Reply:
fixed